### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by Sahar-AbdelSamad**
## In this PR we added validation to the email entered in the form, using JavaScript. We:
- Implemented some interactions:
     - A message appears if the email entered contains capital letters, and the form is not sent
     - The form is sent if the email doesn't contain capital letters
- Styled the new class
- Followed all the requirements

[You can check the project live here](https://sahar-abdelsamad.github.io/Portfolio-setup-and-mobile-version-skeleton/)

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -104,3 +104,33 @@ popupBtn.forEach((item) => { item.addEventListener('click', openPopup); });
...
+const error = document.createElement('small');
+error.className = ('error');
+error.innerText = ('The content of the email field has to be in lower case');
+formMessage.appendChild(error);
+
+const failed = (event) => {
+  event.preventDefault();
+  error.classList.add('wrong-email');
+  error.classList.remove('error');
+};
```
Comment (HENRYKC24) [inline script.js:122]: - [x] I am really impressed by your implementations. You did a great job:+1: and I want you to do even do greater things in this project. Could you, please, add color to your error message that will blend with the colors in the template?
You can choose from the colors below(pulled from the template you are using):point_down:
image

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED:recycle:
Do More
## Hi @Sahar-AbdelSamad and @lisandroseia:wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Sahar-AbdelSamad) [reply script.js:122]: Thank you :+1:  We changed the color of the error message to match the template's colors :smile:

**Review by rustam-tolipov — APPROVED** [review]
## Hi @Sahar-AbdelSamad   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**